templates/terraform/examples/base_configs/test_file.go.erb

```diff
@@ -27,9 +28,10 @@ func TestAcc<%= test_slug -%>(t *testing.T) {
 				Config: testAcc<%= test_slug -%>(acctest.RandString(10)),
 			},
 			{
-				ResourceName:      "<%= "google_#{product.downcase}_#{resource_name.underscore}" -%>.<%= example.primary_resource_id -%>",
-				ImportState:       true,
-				ImportStateVerify: true,
+				ResourceName:            "<%= "google_#{product.downcase}_#{resource_name.underscore}" -%>.<%= example.primary_resource_id -%>",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{<%= ignore_read_all %>},
```

Comment:
it's not really necessary since it's autogenerated anyway, but it might also be nice to only include this line if there's something to be ignored